roles/gpu_operator_run_gpu-burn/tasks/main.yml

```diff
@@ -18,12 +18,22 @@
   register: gpu_burn_gpu_nodes
   failed_when: gpu_burn_gpu_nodes.stdout == ""
 
+- name: Create the gpu-burn namespace and label it as privileged
```

Comment:
Do we need gpu-burn to run as privileged? Shouldn't it be able to use the GPU allocated by the Device Plugin?

Reply:
I would argue that we don't need the gpu-burn at all, since the GPU operator does cuda workload and device-plugin [validations](https://gitlab.com/nvidia/kubernetes/gpu-operator/-/tree/master/validator) as well.

But let me try to change the gpu-burn Pod security context and remove the privileged PSa from the respective namespace and verify that it works.

Reply:
Issue resolved in https://github.com/rh-ecosystem-edge/ci-artifacts/pull/58. Thanks @fabiendupont for the valuable input :+1: